tests/examples/digital_fingerprinting/test_dfp_file_batcher_stage.py

```diff
@@ -118,7 +118,7 @@ def test_on_data_two_batches(config: Config,
     expected_10_26_files = sorted(f.path
                                   for f in fsspec.open_files(os.path.join(test_data_dir, '*_2022-01-30_10-26*.json')))
 
-    (batch1, batch2) = batches
+    (batch1, batch2) = batches[0], batches[1]  # Make pylint happy. It doesn't like ambiguous unpacking
```

Comment:
Oooooph I think we might want to consider adding that rule to the `disable` list in `pyproject.toml`

Reply:
I don't know, this seems to make some sense, since the function call doesn't have to return an unpackable tuple of size 2.